### ISSUE TITLE
Fix Permission after or when install theme? Linux Mint users

### ISSUE THREAD

**Issue by SmileYzn** [vinceliuice/Fluent-gtk-theme#67]
Hi, first thanks for working on this theme.

When you install theme using install.sh need to fix /usr/share/themes permission with chmod to 777 or something else to theme correctly work on linux mint with cinnamon.

also need to fix icon theme at /usr/share/icons, cursors, wallpapers etc.

I hope that help someone that getting problems.

Ps. I think also need to check other vinceluice themes also..

Thanks again

**Comment by vinceliuice** [comment]
Why you have permission issue? just run `./install.sh` to install the theme there should have no such issue

**Comment by SmileYzn** [comment]
When you install as root theme is used in some applications when run with root privleges. Also theme is displayed for all users in machine.

Wihout sudo when install the theme gets installed only on user context.

**Comment by vinceliuice** [comment]
Is there any issue when you run it with sudo？

**Comment by SmileYzn** [comment]
No, Just need to fix /usr/share/themes with chmod -R 777 to get loaded correctly on Linux mint.

**Comment by vinceliuice** [comment]
Why this is needed？ theme only need read authority！

**Comment by vinceliuice** [comment]
Is t linux mint can not use this theme？

**Comment by SmileYzn** [comment]
No, just a bug on cinnamon in Linux mint, that not reading theme correctly even with installed with sudo. I did not know why this happen, but fix setting permission on theme folder.

**Comment by vinceliuice** [comment]
Ok I get it, I'll test on linux mint

**Comment by SmileYzn** [comment]
Thanks to be patient 🙏. I just did not know why this happen there. 

Also I check other themes (including from other devs) and sometimes this happen to.

**Comment by vinceliuice** [comment]
You are welcome, I'll test and find the reason, by the way, zip file unable to save the authorities of Linux files, so when you use the github version I suggest to use git clone it

**Comment by SmileYzn** [comment]
Git clone? Oh ok. 

Maybe fix permission using install script or some kind of alert during install process will be welcome too. 

Man really thanks for check this, I never suspect that permissions can cause some problems at themes.

Ps. Icons and cursor themes can have same problem?

**Comment by vinceliuice** [comment]
Yes zip is for windows, linux should download the tar.bz2 or tar.gz one, this can save all permission with no issue
